### PR TITLE
Return missed paddings

### PR DESCRIPTION
**Pull request by iimironov**

### CODE REVIEW

File: model-optimizer/extensions/back/ConvolutionNormalizer.py [hunk 1]
```diff
@@ -247,9 +247,6 @@ def replace_pattern(self, graph: Graph, match: dict):
             shape_src.connect(ss_0.in_port(0))
             ss_0.out_port(0).connect(node.in_port(2))
 
-            # Specification: *padding amount* is deduced from relation of input and output spatial shapes
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/back/ConvolutionNormalizer.py:250]: Is this correct approach? We have discussed that the fix should be done in the IE IR reader.

@slyalin , @GlebKazantaev what do you think?

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/extensions/back/ConvolutionNormalizer.py (hunk 1 above)
Comment (iimironov) [reply model-optimizer/extensions/back/ConvolutionNormalizer.py:250]: According to conversation with @ilyachur we have to setting all attributes in IR. So we need to fix it on MO side.

Comment (ilyachur) [reply model-optimizer/extensions/back/ConvolutionNormalizer.py:250]: @lazarevevgeny If I remember right we had a discussion, that in IR we should always set all attributes.

Comment (lazarevevgeny) [reply model-optimizer/extensions/back/ConvolutionNormalizer.py:250]: @ilyachur, according to the documentation this attribute must be omitted if the padding type is "auto".

Comment (lazarevevgeny) [reply model-optimizer/extensions/back/ConvolutionNormalizer.py:250]: Just have a quick call with @ilyachur  and he agreed that this is an IE Reader bug and MO should not be changed.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

**Comment by iimironov** [comment]
Will be fixed on IE IR reader side.

**Pull request closed without merging**